### ISSUE TITLE
JAWS does not announce that text has been marked with <del>

### ISSUE THREAD

**Issue by goodwitch** [FreedomScientific/VFO-standards-support#606]
### Summary

1. Go to https://html.com/tags/del/ and listen to the page using JAWS/Chrome.
2. JAWS does not announce that "Encyclopedia Britannica" has a strike through indicating that this content is being deleted.  On this page, the words "Encyclopedia Britannica" are semantically marked with <del> to indicate this content is being deleted. 

### Expected result

JAWS needs to announce the semantics of the <del>.  Example, JAWS needs to say:
The most important encyclopedia in the world is (begin deletion) Encyclopedia Britannica (end deletion) (being insert) Wikipedia (end Insert).

NVDA does announce this semantic content.  VoiceOver on desktop has an earcon for deleted content (unfortunately it is not a default setting in VO on desktop).  I really, really, really think that <del> semantics should be automatically announced by the screen reader (because if it isn't...that oh my goodness, the meaning of that content is NOT being properly conveyed!!!)

### Actual result

JAWS says, "The most important encyclopedia in the world is Encyclopedia Britannica Wikipedia."
(this does NOT convey the content that is being displayed to sighted users and changes the meaning so that the screen reader user does not have equal access).

#### JAWS version and build number
JAWS Version 2022.2110.36 ILM
 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome 98

**Comment by JAWS-test** [comment]
The problem also occurs when I explicitly select in the JAWS Settings Center under "Speech and Sounds Schemes" that deleted or strikethrough text should be read with a hint or a sound.

The problem also occurs when the text is strikethrough with `<s>` or `text-decoration:line-through`.

In contrast, underlined text (`<ins>`, `<u>` or `text-decoration:underline`) is output correctly by JAWS, provided I have configured this in the settings.

Related: https://github.com/w3c/aria/issues/905#issuecomment-1034596365

**Comment by stevefaulkner** [comment]
@ggordon-vispero heads up on this.
